example/dash_analytics_example.dart

```diff
@@ -35,18 +33,23 @@ final Analytics analytics = Analytics(
   tool: tool,
   homeDirectory: getHomeDirectory(),
```

Comment:
@christopherfujino do you suggest that this method that resolves the user's home directory be moved into the package?

Reply:
I think that would be a good idea